src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,27 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeleteHistories {
+    private List<DeleteHistory> deleteHistories = new ArrayList<>();
+
+    public DeleteHistories() {
+    }
+
+    private DeleteHistories(List<DeleteHistory> deleteHistories) {
+        this.deleteHistories = deleteHistories;
+    }
```

Comment:
final과 Collections의 unmodifiableList를 사용해서 일급 컬렉션을 불변 객체로 만들어 보는 것도 좋은 경험이 되실거라 생각합니다.
JPA의 연관관계로 일급 컬렉션을 만들었을때는어쩔 수 없이 불변객체로 사용할 수 없지만, JPA를 의존하지 않는 일급 컬렉션은 충분히 불변객체로 사용이 가능합니다 😄 

불변 객체로 만들게되면 add와 같은 메서드는 사용하실 수 없게 되는데, 생성자를 활용해보는 것도 좋을 것 같아요 🤔 

```suggestion
    private final List<DeleteHistory> deleteHistories;

    private DeleteHistories(List<DeleteHistory> deleteHistories) {
        this.deleteHistories = Collections.unmodifiableList(deleteHistories);
    }
```

Reply:
요한 리뷰어님께서 의도하신지는 모르겠지만, JPA를 사용하는 프로젝트 전반에는 final를 쓰면 안되겠구나 하는 막연함이 있었는데요. 그 구분점을 알려주실려고, 요구사항을 주신 것으로 생각됩니다. 수정후 리뷰 재요청 드리겠습니다.
감사합니다. 😄